lib/cPanel/TQSerializer/YAML.pm

```diff
@@ -1,10 +1,5 @@
 package cPanel::TQSerializer::YAML;
 
-# cpanel - cPanel/TQSerializer/YAML.pm            Copyright(c) 2014 cPanel, Inc.
-#                                                           All rights Reserved.
-# copyright@cpanel.net                                         http://cpanel.net
-# This code is subject to the cPanel license. Unauthorized copying is prohibited
-
 use YAML::Syck ();
 
 # with no bless flags
```

Comment:
I see no mention of the Perl license in this file. Is having it in the separate POD file sufficient?